include/bh_python/register_axis.hpp

```diff
@@ -114,6 +114,41 @@ py::class_<A> register_axis(py::module& m, Args&&... args) {
                  }
              })
 
+        .def_property_readonly("traits_underflow",
+                               [](const A& self) {
+                                   return static_cast<bool>(
+                                       bh::axis::traits::options(self)
+                                       & bh::axis::option::underflow);
+                               })
+        .def_property_readonly("traits_overflow",
+                               [](const A& self) {
+                                   return static_cast<bool>(
+                                       bh::axis::traits::options(self)
+                                       & bh::axis::option::overflow);
+                               })
+        .def_property_readonly("traits_circular",
+                               [](const A& self) {
+                                   return static_cast<bool>(
+                                       bh::axis::traits::options(self)
+                                       & bh::axis::option::circular);
+                               })
+        .def_property_readonly("traits_growth",
+                               [](const A& self) {
+                                   return static_cast<bool>(
+                                       bh::axis::traits::options(self)
+                                       & bh::axis::option::growth);
+                               })
+        .def_property_readonly(
+            "traits_continuous",
+            [](const A& self) { return bh::axis::traits::continuous(self); })
+        .def_property_readonly(
+            "traits_inclusive",
+            [](const A& self) { return bh::axis::traits::inclusive(self); })
```

Comment:
I am not sure whether you need to mirror this in Python. This trait is used internally in Boost.Histogram to make the code a little faster when it is possible.